pkgs/development/python-modules/protobuf/default.nix

```diff
@@ -16,14 +16,14 @@ buildPythonPackage rec {
   propagatedBuildInputs = [ protobuf google_apputils ];
   buildInputs = [ google_apputils pyext ];
 
-  patches = [
+  patches = optional (versionAtLeast python.version "3.7")
```

Comment:
Wouldn't `pythonAtLeast` make more sense here?

Reply:
`isPy37`

Reply:
Thanks! the things you can learn about the python ecosystem ;).